code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2611,3 +2611,45 @@
 		var/color
 		CONVERT_PH_TO_COLOR(exposed_atom.reagents.ph, color)
 		exposed_atom.add_atom_colour(color, WASHABLE_COLOUR_PRIORITY)
+
+/datum/reagent/ants
+	name = "Ants"
+	description = "A sample of a lost breed of Space Ants (formicidae bastardium tyrannus), they are well-known for ravaging the living shit out of pretty much anything."
+	reagent_state = SOLID
+	color = "#993333"
+	taste_mult = 1.3
+	taste_description = "<span class='warning'>ANTS OH GOD</span>"
+	metabolization_rate = 5 * REAGENTS_METABOLISM //1u per second
+	glass_name = "glass of ants"
+	glass_desc = "Bottoms up...?"
+	var/ant_damage = 0
+	var/amount_left = 0
+
+/datum/reagent/ants/on_mob_life(mob/living/carbon/M)
+	M.adjustBruteLoss(max(0.5, round((ant_damage * 0.1),0.1)))
```

Comment:
Assuming a full 50 units of ants are injected or fed to a target, they'll take 103.75 brute damage in 50 seconds. You might to tune this down a bit.

Reply:
😋